### ISSUE TITLE
NVDA and FreeAudio Converter: the object navigation  doesn't work properly

### ISSUE THREAD

**Issue by florianionascu7** [nvaccess/nvda#11010]
Hello! If you use NVDA with Free Audio Converter by DVD Video Soft, which is almost accessible, you can add files, set the folder you want to find them after the conversion and select the format you want them to be converted to, but you won't be able to find the convert button if you navigate with tab and the arrow keys, so you shall use the object navigation to find that option, but you can't activate it, because NVDA treats it as a text object, not as a function object, therefore it says "no action" instead.
Here's a demo recording, where I explained step by step how to reproduce this issue.
The NVDA version I use is 2019.3.1 on Windows 10 version 1909 x64, and the latest version of Free Audio Converter, 5.1.9 - Premium.
[NVDA&FreeAudioConverter.zip](https://github.com/nvaccess/nvda/files/4480535/NVDA.FreeAudioConverter.zip)
[]([NVDA&FreeAudioConverter.zip](https://github.com/nvaccess/nvda/files/4480542/NVDA.FreeAudioConverter.zip))

**Comment by lukaszgo1** [comment]
I don't have experience with this specific software, however if I were you I would try routing  mouse to this object with NVDA+numpad slash and click it with left mouse button. 
Could you also post the developer info for the control in question by moving navigation object into it, pressing NVDA+f1 and posting everything from the log from the caret position to the bottom?

**Comment by florianionascu7** [comment]
It works, many Thanks!

**Comment by Brian1Gaff** [comment]
Might still be worth pointing this non standard control out to the authors, 
as they may be able to fix it simply.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----
